package.json

```diff
@@ -19,8 +20,8 @@
     "@agm/core": "3.0.0-beta.0",
     "@alfresco/adf-core": "4.7.0",
     "@alfresco/adf-extensions": "4.7.0",
-    "@alfresco/js-api": "4.7.0",
-    "@angular-material-extensions/password-strength": "10.0.0",
+    "@alfresco/js-api": "^4.7.0",
+    "@angular-material-extensions/password-strength": "^10.0.0",
```

Comment:
I am not sure Renovate will be happy with these changes (adding the `^`) 🤔
(It is configured to use exact versions when updating dependencies; not sure how it can cope with non-exact versions present in `package.json`.)

Reply:
OK - I'll fix that